content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -190,6 +190,8 @@ The following manifest defines an Ingress that sends traffic to your Service via
 
    {{< note >}}If you are running Minikube locally, use `minikube ip` to get the external IP. The IP address displayed within the ingress list will be the internal IP.{{< /note >}}
 
+   {{< note >}}If you are running Minikube locally on Mac, use `127.0.0.1` and run `minikube tunnel`.{{< /note >}}
```

Comment:
Hi @cgammans . I have not checked the macOS steps. When do you run `minikube tunnel`?

Reply:
@cgammans Please make this note a little bit verbose, i.e. what do you mean by "user `127.0.0.1`, when to run `minikube tunnel`. Or else the note is not that evident for users.

Reply:
Basically, when using a mac, don't uses `minikube ip` as the line above says, but rather just use `127.0.0.1`
It does not say "user 127...", just "use `127..`"